src/LessMsi.Gui/MruMenuStripManager.cs

```diff
@@ -85,9 +86,9 @@ private void LoadPreferences()
         public void SavePreferences()
         {
             PruneItems();
-            var paths = new StringCollection();
-            _items.ForEach((item) => paths.Add(item.FilePathName));
-            Properties.Settings.Default.RecentFiles = paths;
+			ApplicationSettings.Default.RecentFiles.Clear();
+            _items.ForEach((item) => ApplicationSettings.Default.RecentFiles.Add(item.FilePathName));

```

Comment:
Super minor and you don't need to fix it, but keep in mind this `ApplicationSettings.Default.RecentFiles` global state could be left in a undefined state if the foreach operation fails part way through. Generally I think best to prepare the state (e.g. the loop) and then set it a separate operation rather than manipulating global state.

Again, not requiring a change just a best practice to think about IMHO.

Reply:
Will take this into account.